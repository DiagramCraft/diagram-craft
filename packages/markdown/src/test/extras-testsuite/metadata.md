Title: A simple document containing meta-headers
CSS: style.css

Content of the document