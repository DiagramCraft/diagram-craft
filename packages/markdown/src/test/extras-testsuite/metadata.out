<p>Content of the document</p>